prism-clients-generator/src/main/kotlin/io/iohk/atala/prism/generator/Generator.kt

```diff
@@ -141,7 +151,7 @@ class Generator : ServiceGenerator {
             """
                     package $javaPackageName.sync
                     
-                    import io.iohk.atala.prism.protos.${service.name}Coroutine
+                    import $javaPackageName.${service.name}Coroutine
```

Comment:
this is minor fix, it doesn't help, and further investigation of the TODO above is needed. I will create an issue about it.